api-reference/api-reference-guide.md

```diff
@@ -0,0 +1,243 @@
+
+Base URL is a common segment to which the API endpoint paths are appended. Defining the base URL in this section reduces the effort to duplicate the segment in each endpoint reference.
+
+The base URL may differ for different groups of users or in different environments. If multiple base URLs are available, list them in this section and clearly describe the conditions of using each of them.
+
+### About the "Authorization" section
+
+Use this section to specify the authentication and authorization requirements of using the APIs, including the authorization type, request schema, possible error codes, and examples.
+
+:information_source: **Tips**:
+
+- This section should only describe the technical specifications of the API authorization, such as parameters and token expiration. Details of how to authenticate should be documented in a separate how-to guide and be referred to as a link here.
+- Provide an example of the authorization request or command. You can use random strings to replace the actual secrets.
+- If there is more than one authorization type applicable to your APIs, describe and specify the use case of each. If one of the options is preferred, highlight it.
+- If different permission levels apply to different API endpoints, document the requirements in the reference for that endpoint.
+
+### About the "Versioning" section
+
+This section is optional. If applicable, use this section to describe the versioning, compatibility, and lifecycle policies of your API set. If migration guides are available, provide the links here.
+
+
+### About the "Pagination" section
+
+This section is optional. If applicable, use this section to describe the options and default values for dividing long API responses into pages.
+
+:information_source: **Tip**: Your readers might not understand what "pagination" is, so it is wise to embed an explanation in the descriptive text.
+
+### About the "Rate limiting and throttling" section
+
+This section is optional. If applicable, use this section to describe the global rate limiting settings and quota of your APIs. If different quotas apply to different endpoints, also add this section in the reference for endpoints.
+
+
+### About the "HTTP status code" section
+
+Use this section to provide the list of HTTP status code that are applicable to your APIs. It is possible that the APIs only return certain types of the standard HTTP status codes.
+
+:information_source: **Tips**:
+
+- For REST APIs, HTTP status codes are used. If you are writing references for other protocols, adjust the table columns.
+- In the `Description` column, describe the causes of that error, and provide related information or links about how to deal with the error.
+
+### About the "Errors" section
+
+This section is optional. If applicable, use this section to list the custom error types defined for the APIs. For easier navigation, you may provide each of the error definitions as a subsection.
+
+### Additional sections
```

Comment:
**suggestion:** Perhaps other standard sections that could be included before the 'Additional sections' would be:
- License (specifying the license under which the API is released)
- Contact (specifying a contact for questions about the API)